mu-plugins/blocks/local-navigation-bar/postcss/style.pcss

```diff
@@ -16,6 +16,7 @@
 }
 
 .wp-block-wporg-local-navigation-bar {
+	height: var(--wp--custom--local-navigation-bar--spacing--height, auto);
```

Comment:
At the moment we control this height by adding padding. I think if we're going to set a fixed height we should remove that padding and set a default height here. If I do that on Developer, the items fill the height nicely due to the flex layout.
 
```suggestion
	height: var(--wp--custom--local-navigation-bar--spacing--height, 60px);
```

![Screenshot 2023-11-23 at 3 12 51 PM](https://github.com/WordPress/wporg-mu-plugins/assets/1017872/21c32719-4caa-4f9d-a931-977aa8df01f0)

Reply:
I am thinking about keeping 'auto' but we can remove the padding. 

Because I'm looking to the theme itself to decide how much height to use, rather than uniformly fixing it in mu-plugins. Like the height [here](https://github.com/WordPress/wporg-developer/pull/404/files#diff-352bae2bc3c7b2b54e339a2f67279c9ae3fc544c474c8916082c95b3bba4d914R157) for the Developer is 60px, but other themes can define their own. 

That being said, I guess all our themes' heights should be 60px, right? 
In that case, we could just set it to 60px here.

Reply:
Padding has been removed.

Reply:
Nice, yeah setting the default here at 60px means no changes in any of the themes currently using it.